src/api/cvc4cpp.cpp

```diff
@@ -2342,9 +2145,13 @@ OpTerm Solver::mkOpTerm(Kind kind, uint32_t arg)
       res = d_exprMgr->mkConst(CVC4::TupleUpdate(arg));
       break;
     default:
-      // CHECK: kind valid?
-      Assert(!res.isNull());
+      PrettyCheckArgument(true,
```

Comment:
This should be `false` instead of `true`, no?

Reply:
Yes. Again, confusing....